multinet/api.py

```diff
@@ -24,11 +27,62 @@
     AlreadyExists,
     RequiredParamsMissing,
 )
-from multinet.user import current_user
+from multinet.user import current_user, find_user_from_id
 
 bp = Blueprint("multinet", __name__)
 
 
+# Included here due to circular imports
+# TODO: Remove once implementing new ORM and permission storage
+def _permissions_id_to_user(permissons: WorkspacePermissions) -> Dict:
+    """Transform permission documents to directly container user info."""
```

Comment:
The grammar here is a bit off. Should it be `contain`?

Reply:
I think a longer docstring would be useful here. Reading the source code is somewhat confusing due to the types shifting and the structure of our basic data changing. A paragraph following the summary comment you have here would be great because newcomers to the codebase (in this case, me!) would better understand what this utility function is for, and why it's written in this way, etc.

Reply:
c60d404